wad.md

```diff
@@ -82,14 +82,15 @@ WAD versions 2 and 3 include all fields.
 |  12 |    4 | u32    | compressed size                        |
 |  16 |    4 | u32    | uncompressed size                      |
 |  20 |    1 | u8     | compression type                       |
-|  21 |    1 | bool   | *unknown*                              |
-|  22 |    2 | u16    | *unknown*                              |
+|  21 |    1 | bool   | [is duplicated ?](#entry-duplication)  |
+|  22 |    2 | u16    | padding                                |
```

Comment:
Is it really padding (value is actually unusued)?
If yes, is value constant? In this case, it could be documented too.

Reply:
Yeah, this value does not seem to be used at all currently.
The value is 0x0 in all files.